src/NServiceBus.RabbitMQ.AcceptanceTests/NServiceBus.RabbitMQ.AcceptanceTests.csproj

```diff
@@ -21,6 +21,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
I'm not sure about this. I used to do it in other OSS projects but it creates a lot of friction for contributors. This will fail the build and prevent us from merging PR's which have warnings. I prefer to be able to merge a PR with the odd warning or two and fix them immediately afterwards with another trivial PR of my own.